Examples/MAX32655/I2S/main.c

```diff
@@ -74,8 +74,10 @@ int main()
     memset(i2s_rx_buffer, 0, sizeof(i2s_rx_buffer));
 
     /* Configure I2S interface parameters */
-    req.wordSize = MXC_I2S_DATASIZE_WORD;
+    req.wordSize = MXC_I2S_WSIZE_WORD;
     req.sampleSize = MXC_I2S_SAMPLESIZE_THIRTYTWO;
+    req.bitsWord = 32 - 1;
```

Comment:
The I2S drivers can handle the `- 1` for the user because it's a register-specific rule. The user should just focus on what `bitsWord` value they need.

You can reference what we do for setting the [DMA burst size](https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/PeriphDrivers/Source/DMA/dma_reva.c#L209). The user passes in the burst size they want, and the DMA drivers properly write the decremented value to the register.

Reply:
Fixed in https://github.com/Analog-Devices-MSDK/msdk/pull/847/commits/27c49436b024eb2e392c597335da0d02698d3aca